### PR TITLE
additional types for SAP Gateway

### PR DESCRIPTION
**Pull request by gregorwolf**

### CODE REVIEW

**Review by larshp — CHANGES_REQUESTED** [review]
.

**Comment by gregorwolf** [comment]
?

**Comment by gregorwolf** [comment]
Can't see what change you requested.

**Comment by gregorwolf** [comment]
Better?

File: src/gateway/#iwbep#cl_mgw_abs_data.clas.abap [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        get_stream         TYPE string VALUE 'GET_STREAM',
+        update_stream      TYPE string VALUE 'UPDATE_STREAM',
+      END OF gcs_methods.
+
+ENDCLASS.
+
+CLASS /iwbep/cl_mgw_abs_data IMPLEMENTATION.
+  METHOD /iwbep/if_mgw_appl_srv_runtime~get_stream.
+
+    RAISE EXCEPTION TYPE /iwbep/cx_mgw_tech_exception
```
Comment (larshp) [inline src/gateway/#iwbep#cl_mgw_abs_data.clas.abap:20]: just leave the method implementations blank or "RETURN.", its not needed for linting

**Review by larshp — CHANGES_REQUESTED** [review]
see above

**Comment by larshp** [comment]
closing

**Pull request closed without merging**